rakelib/default_plugins.rb

```diff
@@ -119,6 +119,8 @@ module RakeLib
       logstash-input-stdin
       logstash-input-tcp
       logstash-output-stdout
+      logstash-output-null
```

Comment:
Sorry to say, we could not as we need to have this as runtime dependencies because of the plugins system.